[2024-10-24T06:55:21.398+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-24T06:55:21.412+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test2.run_selenium_task manual__2024-10-24T06:55:16.057524+00:00 [queued]>
[2024-10-24T06:55:21.420+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test2.run_selenium_task manual__2024-10-24T06:55:16.057524+00:00 [queued]>
[2024-10-24T06:55:21.420+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-24T06:55:21.437+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): run_selenium_task> on 2024-10-24 06:55:16.057524+00:00
[2024-10-24T06:55:21.442+0000] {standard_task_runner.py:72} INFO - Started process 673 to run task
[2024-10-24T06:55:21.447+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'test2', 'run_selenium_task', 'manual__2024-10-24T06:55:16.057524+00:00', '--job-id', '868', '--raw', '--subdir', 'DAGS_FOLDER/dcard_dag2.py', '--cfg-path', '/tmp/tmppsqoq0um']
[2024-10-24T06:55:21.450+0000] {standard_task_runner.py:105} INFO - Job 868: Subtask run_selenium_task
[2024-10-24T06:55:21.499+0000] {task_command.py:467} INFO - Running <TaskInstance: test2.run_selenium_task manual__2024-10-24T06:55:16.057524+00:00 [running]> on host 60bbe790d101
[2024-10-24T06:55:21.576+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='test2' AIRFLOW_CTX_TASK_ID='run_selenium_task' AIRFLOW_CTX_EXECUTION_DATE='2024-10-24T06:55:16.057524+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-24T06:55:16.057524+00:00'
[2024-10-24T06:55:21.577+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-24T06:55:21.598+0000] {logger.py:11} INFO - ====== WebDriver manager ======
[2024-10-24T06:55:21.699+0000] {logger.py:11} INFO - Get LATEST chromedriver version for google-chrome
[2024-10-24T06:55:21.829+0000] {logger.py:11} INFO - Get LATEST chromedriver version for google-chrome
[2024-10-24T06:55:21.964+0000] {logger.py:11} INFO - Driver [/home/airflow/.wdm/drivers/chromedriver/linux64/130.0.6723.69/chromedriver] found in cache
[2024-10-24T06:55:44.005+0000] {logger.py:11} INFO - ====== WebDriver manager ======
[2024-10-24T06:55:44.105+0000] {logger.py:11} INFO - Get LATEST chromedriver version for google-chrome
[2024-10-24T06:55:44.225+0000] {logger.py:11} INFO - Get LATEST chromedriver version for google-chrome
[2024-10-24T06:55:44.344+0000] {logger.py:11} INFO - Driver [/home/airflow/.wdm/drivers/chromedriver/linux64/130.0.6723.69/chromedriver] found in cache
[2024-10-24T06:55:52.366+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-10-24T06:55:52.371+0000] {process_utils.py:132} INFO - Sending 15 to group 673. PIDs of all processes in the group: [702, 937, 943, 952, 953, 977, 979, 1018, 1027, 972, 708, 717, 718, 742, 805, 744, 783, 792, 869, 737, 673]
[2024-10-24T06:55:52.372+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 673
[2024-10-24T06:55:52.374+0000] {taskinstance.py:3092} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-10-24T06:55:52.388+0000] {taskinstance.py:3310} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/dcard_dag2.py", line 332, in selenium_scraper
    sleep(10)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3094, in signal_handler
    raise AirflowTaskTerminated("Task received SIGTERM signal")
airflow.exceptions.AirflowTaskTerminated: Task received SIGTERM signal
[2024-10-24T06:55:52.405+0000] {taskinstance.py:1225} INFO - Marking task as UP_FOR_RETRY. dag_id=test2, task_id=run_selenium_task, run_id=manual__2024-10-24T06:55:16.057524+00:00, execution_date=20241024T065516, start_date=20241024T065521, end_date=20241024T065552
[2024-10-24T06:55:52.428+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=943, status='terminated', started='06:55:43') (943) terminated with exit code None
[2024-10-24T06:55:52.432+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-24T06:55:52.481+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=742, status='terminated', started='06:55:21') (742) terminated with exit code None
[2024-10-24T06:55:52.482+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=979, status='terminated', started='06:55:43') (979) terminated with exit code None
[2024-10-24T06:55:52.482+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=977, status='terminated', started='06:55:43') (977) terminated with exit code None
[2024-10-24T06:55:52.483+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=744, status='terminated', started='06:55:21') (744) terminated with exit code None
[2024-10-24T06:55:52.483+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=1018, status='terminated', started='06:55:44') (1018) terminated with exit code None
[2024-10-24T06:55:52.484+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=792, status='terminated', started='06:55:21') (792) terminated with exit code None
[2024-10-24T06:55:52.484+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=937, status='terminated', started='06:55:43') (937) terminated with exit code None
[2024-10-24T06:55:52.484+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=737, status='terminated', started='06:55:21') (737) terminated with exit code None
[2024-10-24T06:55:52.485+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=717, status='terminated', started='06:55:21') (717) terminated with exit code None
[2024-10-24T06:55:52.485+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=869, status='terminated', started='06:55:33') (869) terminated with exit code None
[2024-10-24T06:55:52.485+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=783, status='terminated', started='06:55:21') (783) terminated with exit code None
[2024-10-24T06:55:52.486+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=952, status='terminated', started='06:55:43') (952) terminated with exit code None
[2024-10-24T06:55:52.486+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=708, status='terminated', started='06:55:21') (708) terminated with exit code None
[2024-10-24T06:55:52.486+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=1027, status='terminated', started='06:55:44') (1027) terminated with exit code None
[2024-10-24T06:55:52.487+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=673, status='terminated', exitcode=2, started='06:55:20') (673) terminated with exit code 2
[2024-10-24T06:55:52.487+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=805, status='terminated', started='06:55:22') (805) terminated with exit code None
[2024-10-24T06:55:52.488+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=702, status='terminated', started='06:55:21') (702) terminated with exit code None
[2024-10-24T06:55:52.488+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=953, status='terminated', started='06:55:43') (953) terminated with exit code None
[2024-10-24T06:55:52.488+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=972, status='terminated', started='06:55:43') (972) terminated with exit code None
[2024-10-24T06:55:52.489+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=718, status='terminated', started='06:55:21') (718) terminated with exit code None
